.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
This has worked well for me in other non-spinnaker repositories. 
Be good to add to all of the repositories.

Weekly interval might generate a lot of PR's (especially across all repositories).
See: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval